.npmignore

```diff
@@ -1 +1,32 @@
 node-tests/
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+FEATURES.md
+STYLEGUIDE.md
+VERSION
+bin
+blueprints
+bower.json
+bower_components
+config
+# dist
+ember-cli-build.js
+ember-source-2.10.0-alpha.1.tgz
```

Comment:
Should this be something like `ember-source-*.tgz` or will be necessary to update after each version?